tmt/steps/execute/internal.py

```diff
@@ -25,8 +27,87 @@
 from tmt.steps.provision import Guest
 from tmt.utils import EnvironmentType, Path, ShellScript, field
 
+TEST_PIDFILE_DIRECTORY = tmt.utils.effective_workdir_root()
```

Comment:
Shouldn't there be `effective_pidfile_root()` instead of `tmt.utils.effective_workdir_root()`? Also, if I'm not mistaken, the variable `TEST_PIDFILE_DIRECTORY` is not used.

Reply:
Correct, hopefully fixed in 985fdde1